spec/jsdoc-spec.coffee

```diff
@@ -936,6 +936,36 @@ describe "JSDoc grammar", ->
       expect(tokens[7]).toEqual value: '}', scopes: ['source.js', 'comment.block.documentation.js', 'entity.name.type.instance.jsdoc', 'punctuation.definition.bracket.curly.end.jsdoc']
       expect(tokens[9]).toEqual value: 'variable', scopes: ['source.js', 'comment.block.documentation.js', 'variable.other.jsdoc']
 
+      {tokens} = grammar.tokenizeLine('/** @param {\nnumber\n} variable this is the description */')
```

Comment:
Instead of using `\n`, could you please use `grammar.tokenizeLines` and multiline quotes?  See https://github.com/atom/language-javascript/blob/master/spec/jsdoc-spec.coffee#L1300 for an example.